sla/resources/queries/sla/PlandProtocols.sql

```diff
@@ -1,42 +1,86 @@
-
 SELECT a.project as ProjectID,
-aa.species,
-a.account as Alias,
-y.grantNumber as OGAGrantNumber,
-a.protocol as ParentIACUC,
-a.title as Title,
-a.name as IACUCCode,
-a.startdate as StartDate,
-a.enddate as EndDate,
-i.FirstName,
-i.LastName,
-i.Division,
-p.external_id,
-i.LastName + ': ' + a.name + '('+ p.external_id +')' + ' - ' + a.title + ' (Species: ' + aa.species + ')' as PIIacuc
+       aa.species,
+       a.account as Alias,
+       y.grantNumber as OGAGrantNumber,
+       a.protocol as ParentIACUC,
+       a.title as Title,
+       a.name as IACUCCode,
+       a.startdate as StartDate,
+       a.enddate as EndDate,
+       i.FirstName,
+       i.LastName,
+       i.Division,
+       p.external_id,
+       i.LastName + ': ' + a.name + '('+ p.external_id +')' +  ' Group Id & Name - ' + aa. Group_Id + ', '+ aa.Group_Name + ' - ' + a.title + ' (Species: ' + aa.species + ')' as PIIacuc
 FROM Site.{substitutePath moduleProperty('EHR','EHRStudyContainer')}.ehr.project a
 LEFT JOIN Site.{substitutePath moduleProperty('EHR','EHRStudyContainer')}.ehr.protocol p ON p.protocol = a.protocol
-LEFT JOIN onprc_ehr.investigators i ON i.rowId = a.investigatorId
-LEFT JOIN Site.{substitutePath moduleProperty('EHR','EHRStudyContainer')}.sla.allowableAnimals aa ON a.protocol = aa.protocol
-LEFT JOIN Site.{substitutePath moduleProperty('ONPRC_Billing','BillingContainer_Public')}.onprc_billing_public.aliases y ON y.alias = a.account
+    LEFT JOIN onprc_ehr.investigators i ON i.rowId = a.investigatorId
+    LEFT JOIN Site.{substitutePath moduleProperty('EHR','EHRStudyContainer')}.sla.allowableAnimals_BreedingGroups aa ON a.protocol = aa.protocol
```

Comment:
Where is this query or table defined? Our automated tests aren't finding it defined anywhere. I don't see it on PRIMe itself either.

Reply:
https://prime20test.ohsu.edu/query/ONPRC/EHR/begin.view?#sbh-qdp-%26sla%26allowableAnimals_BreedingGroups